### PR TITLE
[semver:minor] Add circleci_ip_range parameter

### PR DESCRIPTION
**Pull request by MichaelAmatoHelix**

### CODE REVIEW

**Review by corrjo — APPROVED** [review]

File: src/jobs/test-v2.yml [hunk 1]
```diff
@@ -26,6 +26,12 @@ parameters:
     default: xlarge
     description: Type of the resource class
     type: string
+  use_circleci_ip_ranges:
+    description: "Whether to use the circle ci ip ranges."
+    type: boolean
+    default: false
+
+circleci_ip_ranges: << parameters.use_circleci_ip_ranges >>
```
Comment (JackyLiHelix) [inline src/jobs/test-v2.yml:34]: Is this attempting to allow CircleCI Docker Cloud runners to access our private VPC to connect to RDS database or similar?

Comment (MichaelAmatoHelix) [reply src/jobs/test-v2.yml:34]: no, I have IP filtering in place on my APIs (registered w/ netskope private access for internal access) but want to be able to reach the API via circle ci runners for automation tests.

Comment (MichaelAmatoHelix) [reply src/jobs/test-v2.yml:34]: (sorry, I didn't see this comment until after I had clicked 'merge', I would not have merged if I had known there were outstanding questions/concerns)

Comment (JackyLiHelix) [reply src/jobs/test-v2.yml:34]: All good. As discussed in Slack DM. Concerns were addressed

**Pull request merged**